lighthouse-cli/index.js

```diff
@@ -196,6 +196,13 @@ function lighthouseRun(addresses) {
       }
 
       return lighthouseRun(addresses);
+    })
+    .catch(err => {
```

Comment:
unfortunately I can't remember at all now, but why is this `catch` necessary and errors won't be handled by `handleError`?